Windows.Forms/Controls/ThemedLabel.cs

```diff
@@ -194,7 +202,47 @@ protected override void OnPaint(PaintEventArgs e)
 					if (theme != null && ThemingSupported)
 					{
 						if (rtl == RightToLeft.Yes) tff |= TextFormatFlags.RightToLeft;
-						if (GlowingText)
+
+						if (GlowingText && Environment.OSVersion.Version.Major == 10 && !SystemInformation.HighContrast && !DesignMode)
+						{
+							Color textColor = SystemColors.ControlText;
+
+							// SystemColors.ActiveCaption always returns an ugly shade of blue. SystemColors.ActiveCaptionText returns black.
+							// We must therefore ask DWM for the title bar color. However, the user has the option to use white title bars,
+							// which we must check for first. (I don't know any public API calls that return this value.)
+							using (var key = Registry.CurrentUser.OpenSubKey(@"Software\Microsoft\Windows\DWM"))
+							{
+								var prevalenceValue = key.GetValue("ColorPrevalence", null);
+								if (prevalenceValue != null && Convert.ToBoolean(prevalenceValue))
```

Comment:
Will you do a bit of error checking to ensure you have a value that will cast to Boolean? Maybe:
`prevalenceValue is int i && Convert...`